virttest/qemu_devices/qcontainer.py

```diff
@@ -3201,3 +3201,27 @@ def numa_hmat_cache_define_by_params(self, nodeid, params):
                     aobject=aobject, backend='hmat_type'))
 
         return hmat_cache_devs
+
+    def secret_object(self, secret_id, data, secret_format=None, inline=True):
```

Comment:
How about  `secret_object_define_by_params` ?  Same format with other fuctions.

Others LGTM

Reply:
> How about `secret_object_define_by_params` ? Same format with other fuctions.

Thanks @yanan-fu Actually from the meaning it should be `secret_object_define_by_variables`. If you check the above definitions, some devices have `xxx_define_by_variables` and `xxx_define_by_params`, `xxx_define_by_params` take params object(dictionary) and convert params to some variables then calling `xxx_define_by_variables`.
At the beginning I use `xxx_define_by_variables` but finally I think not necessary since only this one function so do not need to distinguish, if the `secret_object` seems not very objective, what do you think about `generate_secret_object`, or you prefer `xxx_define_by_variables`? I can make the change accordingly.

Reply:
I prefer `xxx_define_by_variables`  a bit more ~ Thanks!